src/OnlineSales/DTOs/ImageDtos.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="ImageDtos.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+using OnlineSales.CustomAttributeValidations;
+using OnlineSales.CustomValidations;
+
+namespace OnlineSales.DTOs
+{
+    public class ImageDtos
+    {
+        [Required]
+        [FileSizeValidateAttribute(10485760, ErrorMessage = "Max file size exceeded")]
```

Comment:
@WASandaruwan where did you get that magic number? Why it it in the code? Any sort of configuration like you have here should be placed in the configuration file and not just be hardcoded.